src/export/ValueSetExporter.ts

```diff
@@ -130,6 +137,174 @@ export class ValueSetExporter {
+      }
+
+      // what we like best of all is when we just have a list of concepts. they'll even have a system specified already!
+      if (errors.length) {
+        logger.error(`Unable to expand ValueSet ${valueSet.name}:\n  - ${errors.join('\n  - ')}`);
+      } else {
+        valueSet.expansion.contains = [];
+
+        valueSet.compose.include.forEach(compose => {
+          // add specific included codes
+          compose.concept?.forEach(concept => {
+            const containedItem: ValueSetExpansionContains = {
+              system: compose.system,
+              code: concept.code
+            };
+            if (compose.version) {
+              containedItem.version = compose.version;
+            }
+            if (concept.display) {
+              containedItem.display = concept.display;
+            }
+            if (concept.designation) {
+              containedItem.designation = cloneDeep(concept.designation);
+            }
+            // add only if the concept is not already present
+            if (!this.isConceptPresent(containedItem, valueSet.expansion.contains)) {
+              valueSet.expansion.contains.push(containedItem);
+            }
+          });
+          // add known included code systems
+          if (!compose.concept?.length && compose.system) {
+            const codeSystem = referencedCodeSystems[compose.system];
+            const newConcepts = this.getConcepts(codeSystem, codeSystem.concept);
+            // flatten so we can remove each existing concept before adding
+            const existingConceptsFlat = flatMap(valueSet.expansion.contains, extractContained);
+            // remove each existing concept from the set of new concepts, to avoid duplication
+            existingConceptsFlat.forEach(existingConcept => {
+              this.removeConcept(existingConcept, newConcepts);
+            });
+            // add whatever remains
+            valueSet.expansion.contains.push(...newConcepts);
+          }
+        });
+        valueSet.compose.exclude?.forEach(compose => {
+          // remove specific excluded codes
+          compose.concept?.forEach(concept => {
+            this.removeConcept(
+              { system: compose.system, code: concept.code },
+              valueSet.expansion.contains
+            );
+          });
+          // remove known excluded code systems
+          // this is a little unusual to do, but we'll allow it.
+          if (!compose.concept?.length && compose.system) {
+            const codeSystem = referencedCodeSystems[compose.system];
+            const conceptsToRemove = flatMap(this.getConcepts(codeSystem, codeSystem.concept));
+            conceptsToRemove.forEach(concept =>
+              this.removeConcept(concept, valueSet.expansion.contains)
+            );
+          }
+        });
+        // Now that we're done, let's add a timestamp and a total
+        valueSet.expansion.timestamp = `${new Date()}`;
```

Comment:
This appears to add a date with a format that looks like:
```
"timestamp": "Tue Mar 02 2021 14:12:53 GMT-0500 (Eastern Standard Time)",
```
The type of `timestamp` is a `dateTime`, and I don't think this string fits the expected format for a `dateTime` in FHIR. Here is my [reference](https://www.hl7.org/fhir/datatypes.html#dateTime) for that. Looks like we will want to switch to the `YYYY-MM-DDThh:mm:ss+zz:zz` format.

Reply:
I think you can just do `(new Date()).toISOString()` -- don't worry about trying to use the local timezone; UTC is fine.

Reply:
And maybe update one of the tests to check the format to make sure it is correct (probably using a regex).